README.md

```diff
@@ -9,6 +9,8 @@
 ![Develop Branch Build](https://img.shields.io/circleci/build/github/kedro-org/kedro/develop?label=develop)
 [![Documentation](https://readthedocs.org/projects/kedro/badge/?version=stable)](https://docs.kedro.org/)
 [![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/6711/badge)](https://bestpractices.coreinfrastructure.org/projects/6711)
+[![Downloads](https://static.pepy.tech/badge/kedro/month)](https://pepy.tech/project/kedro)
+[![Downloads](https://static.pepy.tech/badge/kedro)](https://pepy.tech/project/kedro)
```

Comment:
Does it make sense to specify that the bottom number is total Downloads?

Reply:
Right, would be cool to change this to "Monthly downloads" and "Total downloads"